client/src/app/+my-account/my-account-settings/my-account-video-settings/my-account-video-settings.component.html

```diff
@@ -15,6 +15,18 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <label i18n for="webTorrentPolicy">Policy regarding P2P technologies</label>
+
+    <div class="peertube-select-container">
+      <select id="webTorrentPolicy" formControlName="webTorrentPolicy">
+        <option i18n value="enable">Enable WebTorrent</option>
+        <option i18n value="disable">Disable WebTorrent globally</option>
+        <option i18n value="disable_on_mobile" disabled>Disable WebTorrent on mobile devices (not yet available)</option>
```

Comment:
I think you can remove this, since we will try to do it automatically (see https://github.com/Chocobozzz/PeerTube/issues/95#issuecomment-338431521)

So you would just have a boolean.